apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -706,7 +706,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
       // If the modal function editor is enabled for this level and
       // the dcdo flag is on, initialize the modal function editor.
       blocklyWrapper.functionEditor = new FunctionEditor();
-      blocklyWrapper.functionEditor.init(opt_options);
+      blocklyWrapper.functionEditor.init(options);
```

Comment:
If we pass options instead, that fixes the rendering issue, because the default renderer is in `options` not `opt_options`, see: https://github.com/code-dot-org/code-dot-org/pull/54080/files#diff-39f0e440025c4f23f4050e7fb36db18df364b1e6d5cf578b326ca71720c8c75bL626